Library/Formula/smpeg2.rb

```diff
@@ -0,0 +1,46 @@
+class Smpeg2 < Formula
+  desc "SDL MPEG Player Library"
+  homepage "http://icculus.org/smpeg/"
+  url "svn://svn.icculus.org/smpeg/tags/release_2_0_0/"
+
+  head "svn://svn.icculus.org/smpeg/trunk"
+
+  option "with-plaympeg", "Build plaympeg application"
+  if build.with? "plaympeg"
+    conflicts_with "homebrew/head-only/smpeg", :because => "both install `plaympeg` binaries."
```

Comment:
We don't do cross-tap `conflicts` in the core.

There's no guarantee the user has `head-only` tapped and when they do `brew install smpeg2 --with-plaympeg` it'll look for a `head-only` formula which won't exist if `head-only` isn't tapped and break.

The conflict would need adding to the `head-only` formula.